SECURITY.md

```diff
@@ -0,0 +1,60 @@
+# Security Process and Policy
+
+This document provides the details on the Veriason security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the Veraison organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the Veraison community. All reports are thoroughly investigated by a set of Veraison maintainers.
+
+To make a report please email the private security list at ___@___ with the details.
```

Comment:
Unfortunately, we only have scope to add Arm addresses at the moment. As Thomas points out, when we have access to CCC hosted resources this can change.

Reply:
We probably want an email, not tied to any one vendor. Dare I suggest an outlook.com or gmail id until the CCC officially adopts veraison?

Reply:
Updated with a google distribution list email